qiskit/circuit/library/standard_gates/global_phase.py

```diff
@@ -34,13 +34,15 @@ class GlobalPhaseGate(Gate):
             \end{pmatrix}
     """
 
-    def __init__(self, phase: ParameterValueType, label: Optional[str] = None):
+    def __init__(
+        self, phase: ParameterValueType, label: Optional[str] = None, duration=None, unit=None
+    ):
```

Comment:
Since we're defining new keyword arguments, we have the option to make them keyword-only (so you _can't_ pass them positionally like `XGate("my label", 1.0, "dt")` but have to say `XGate(duration=1.0, unit="dt")`).  We can put a `*` between the last allowed-to-be positional argument and the first kwarg, e.g.
```suggestion
    def __init__(
        self, phase: ParameterValueType, label: Optional[str] = None, *, duration=None, unit=None
    ):
```

In #11014 we're modifying all the singleton standard gate signatures to have that, so it'd be good to have it for these as well.